Snake/BusinessLayer/maze.cs

```diff
@@ -27,17 +27,24 @@ public Maze(int w, int h, int mazeMode)
 
             switch (mazeMode)
             {
-                case (int)MazeLevel.Easy:
+
+                case (int)MazeLevel.Beginner:
                     if (randomNumber.NextDouble() < 0.5)
                     {
-                        GenerateHorizonJungleMaze();
+                        GenerateSuperEasyMaze();
                     }
                     else
                     {
-                        GenerateVerticalJungleMaze();
+                        GenerateSuperEasyMaze();
                     }
                     break;
```

Comment:
No need to use probability to control the generateSuperEasyMaze(). 
The probability should be used for the generateVerticalJungleMaze() and GenerateHorizonJungleMaze().